js/jquery.ntm.js

```diff
@@ -30,6 +31,7 @@
                 if (options.autoParentDetection) {
                     if (item.has('ul')[0]) {
                         item.addClass(options.parentClass);
+                        item.attr('id',"node-"+num); //Adding id to all tree elements

```

Comment:
please lead a space after the comma, space char before and after the +.
We need to please follow coding guidelines.